README.md

```diff
@@ -208,3 +208,8 @@ participated in that process.
 [meeting notes]: ./meetings
 [project overview]: ./docs/project-overview.md
 [query issues for "meeting"]: https://github.com/nodejs/nodejs.dev/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+Meeting+in%3Atitle
+
+### Thanks to all the contributors ❤
```

Comment:
This section should be done before the links in the end of the file.

Reply:
@ovflowd Done! Please re-review this PR.